doc/sphinx/source/n3fit/hyperopt.rst

```diff
@@ -372,6 +378,94 @@ The figure of merit will be the difference between the :math:`\chi2` of the seco
    L_{\rm hyperopt} = \chi^{2}_{(1) \rm pdferr} - \chi^{2}_{(2)}
 
 
+New hyperoptimization metrics with fold and replica statistics
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The combination of :ref:`k-folding <hyperkfolding-label>` and multi-replica experiments
```

Comment:
Maybe provide a brief intro in what you mean here
> The combination of :ref:`k-folding <hyperkfolding-label>` and multi-replica experiments

Perhaps something like the following:
> The hyperopt measures discussed above are all based on performing a single replica fit per fold. However one may also wish to run the hyperoptimization algorithm on fits consisting of many replicas per fold. This is a feasible option in ``n3fit``, since it has been optimised to efficiently run many replica fits in parallel on GPU.

Reply:
Done. Thanks!